tests/unit/models/transactions/test_did_set.py

```diff
@@ -62,3 +62,25 @@ def test_empty(self):
                 error.exception.args[0],
                 "{'did_set': 'Must have one of `did_document`, `data`, and `uri`.'}",
             )
+
+    def test_all_empty_fields(self):
+        with self.assertRaises(XRPLModelException):
+            DIDSet(account=_ACCOUNT, data="", did_document="", uri="")
+
+    def test_delete_multiple_fields(self):
```

Comment:
I would keep the wording consistent with "empty fields" and not use "delete" since this is a unit test.

Reply:
updated in [1b7400a](https://github.com/XRPLF/xrpl-py/pull/717/commits/1b7400a0f12a732cfaf79f521cf69c3199b33c4f)

Reply:
I would use "empty fields" rather than "remove" since this is a unit test.